src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountDocumentServiceImpl.java

```diff
@@ -147,6 +149,14 @@ protected String getACHTransactionCode(String workdayAccountType) {
         }
     }
     
+    protected String determineStandardEntryClass(String achTransactionCode) {
+        if (StringUtils.isNotBlank(achTransactionCode)) {
+            return (StandardEntryClassValuesFinder.StandardEntryClass.valueOf((StringUtils.right(achTransactionCode, 3)).toUpperCase())).toString();
+        } else {
+            throw new IllegalArgumentException("Unrecognized ACH transaction code from file: " + achTransactionCode);
```

Comment:
Since this line only seems to be reachable when the variable in question is blank, you may want to restructure the exception message to avoid confusion. The same goes for the related line in the PaymentWorks service.